.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/*
```

Probably, you need here additional line
```suggestion
logs/*

```